.github/workflows/erlang.yml

```diff
@@ -2,18 +2,23 @@ name: Erlang CI
 
 on:
   push:
-    branches: [ main ]
   pull_request:
-    branches: [ main ]
+    branches: [ $default-branch ]
+  release:
+    types:
+      - created
 
 jobs:
 
   build:
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        otp: [21.3, 22.0.7, 22.3, 23.0.4, 23.2.7.0]
```

Comment:
This is fine for now, but we don't officially support anything below `22`. If, at some point, tests start failing on `21.3`, we'll not fix the system, we will just remove `21.3` from here.

Reply:
Maybe make sense already remove `21.3` and provide some info about minimum support version in `README.md` based on `matrix -> otp` section in `erlang.yml`?